test/cachex/actions/fetch_test.exs

```diff
@@ -28,6 +30,11 @@ defmodule Cachex.Actions.FetchTest do
     fb_opt3 = &{:ignore, String.reverse(&1)}
     fb_opt4 = fn -> "6yek" end
 
+    fb_opt5 = fn ->
+      send(test_process, {callers_reference, Process.get(:"$callers")})
```

Comment:
We can use `callers()` instead of `Process.get(:"$callers")` here.

Reply:
On second thought, I would rather not change this test case at all -- and add a separate test case covering the passthrough of `:"$callers"`.